src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -260,15 +260,36 @@ object DemuxFastqs {
       |    --metadata SampleSheet.csv --metrics metrics.txt --output output_folder
       |```
       |
-      |## Illumina FASTQ File Naming
+      |## Output Standards
```

Comment:
@tfenne I originally authored this, so could you take a look to see if makes sense?

Some context: the `--illumina-standards` option was added a long time ago so that the output FASTQs would follow the format so that they can be uploaded to BaseSpace.  This included the file names AND the header format.  But in some docs for BaseSpace now, the last field in the comment is the sample barcode (index) while previously it was sample number.  

With the three new options, we've deprecated the above option in favor of three explicit options for clearer behavior.